org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/models/AbstractPointsModel.java

```diff
@@ -103,6 +104,6 @@ public boolean equals(Object obj) {
 		} catch (Exception ne) {
 			return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)